src/main/java/com/limechain/storage/block/BlockHandler.java

```diff
@@ -34,10 +37,9 @@ public BlockHandler(ProtocolRequester requester, RuntimeBuilder builder) {
         asyncExecutor = AsyncExecutor.withPoolSize(10);
     }
 
-    public synchronized void handleBlock(Instant arrivalTime, BlockHeader header) {
+    public synchronized void handleBlockNoBody(Instant arrivalTime, BlockHeader header) {
```

Comment:
This function name could be `handleBlockHeader` and the other one could become `handleBlock`